app/views/candidate_interface/application_form/_review.html.erb

```diff
@@ -31,7 +31,7 @@
 <section class="govuk-!-margin-bottom-8">
   <h2 class="govuk-heading-m govuk-!-font-size-27"><%= t('section_groups.personal_details') %></h2>
   <h3 class="govuk-heading-m"><%= t('page_titles.personal_information') %></h3>
-  <%= render(CandidateInterface::PersonalDetailsReviewComponent.new(application_form: application_form, editable: editable, missing_error: missing_error)) %>
+  <%= render(CandidateInterface::PersonalDetailsReviewComponent.new(application_form: application_form, editable: editable, missing_error: missing_error, return_to_application_review: true)) %>
```

Comment:
This feels a bit gross - we have to pass a flag down to the `PersonalDetailsReviewPresenter` which is responsible for populating the `return_to` params in the change link as the PersonalDetailsReview component is used both on the application page and the application review page - but not sure if there is an easier way

Reply:
The additional arg seems fine to me, and could always be switched from a boolean to a string/symbol later on if there are ever more than two possible return paths.

I wonder, though, if this is a good opportunity to collapse the presenter into PersonalDetailsReviewComponent? Doesn't seem to be much point in keeping them separate and might be easier to follow.

Reply:
Had a look into this also, and from what I can see, the reason for the separation may have been to make testing easier (the presenter makes good use of dependency injection allowing for multiple edge cases to be tested). Methinks collapsing will result in result in quite a cumbersome / complex test suite.